docs/user-guide/configuring.md

```diff
@@ -550,84 +550,125 @@ The complete configuration hierarchy, from highest precedence to lowest preceden
 
 ## Extending Configuration Files
 
-If you want to extend a specific configuration file, you can use the `extends` property and specify the path to the file. The path can be either relative or absolute.
+A configuration file can "extend" one or more "child" configurations to override a "base" set of rules.
 
-Configurations can be extended by using:
+The `extends` property value is either a string or an array of strings (in which each additional configuration overrides the preceding configurations) that can specify any of the following:
 
-1. YAML file
-1. JSON file
-1. JS file
-1. Shareable configuration package
+* the recommended rules
+* shareable configuration packages
+* configurations from plugins
+* configuration files
+* all core rules
 
-The extended configuration provides base rules, which can be overridden by the configuration that references it. For example:
+If a "child" configuration has an `extends` property, ESLint extends the configuration recursively.
 
-```js
-{
-    "extends": "./node_modules/coding-standard/.eslintrc",
+The `rules` property can do any of the following to override the "base" set of rules:
+
+* specify different options for rules
+* turn off rules
+* turn on additional rules
+
+
+### Extending the recommended rules
```

Comment:
I think we need to work on the terminology for these headings. "Extending X" is a bit unclear to me. I'd rather saying "Using the recommended rules" and "Using a shareable configuration package"